### ISSUE TITLE
Why is play/stop binded to F11

### ISSUE THREAD

**Issue by DeadbushCarrot** [fudgeu/playlist#52]
Quick and simple F11 makes the fullscreen I think some better bindings should be F6 F7 F8 Same order for functions

**Comment by Felix14-v2** [comment]
Even a better binds:
![image](https://github.com/fudgeu/playlist/assets/75726196/dd33bc4d-bdd4-4dd7-a795-5a4c1273069b)

These are the media keys on keyboard (⏮⏯⏭)

**Comment by DeadbushCarrot** [comment]
> Even a better binds: ![image](https://user-images.githubusercontent.com/75726196/242990892-dd33bc4d-bdd4-4dd7-a795-5a4c1273069b.png)
> 
> These are the media keys on keyboard (⏮⏯⏭)

Not every has media keys plus its way more convenient to use for me I have to hold down FN cause I have 80%/ TKL keyboard then press the button way slower and less convenient

**Comment by Felix14-v2** [comment]
At least it would be more like other media players behave

**Comment by fudgeu** [comment]
I think the defaults are supposed to be bound to '0, -, =' on the keyboard - which I did so specifically because I actually work on a 65% that doesn't have dedicated function keys, lol.

I imagine if you don't have an English/US keyboard layout however, '0, -, and =' might not make a lot of sense and might even cause some weirdness - but it is a little hard to cater to every keyboard out there.

Try pressing 'Reset Keys' if you don't mind, and let me know if it still defaults to F11 or not

Thanks!

**Comment by DeadbushCarrot** [comment]
Yeah the defaults are 0 - = I guess I forgot I changed them or mod conflict
